src/react-app-env.d.ts

```diff
@@ -33,3 +33,16 @@ interface Window {
 declare module "@metamask/jazzicon" {
   export default function (diameter: number, seed: number): HTMLElement
 }
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+type ObjectKeys<T> = T extends object
+  ? (keyof T)[]
+  : T extends number
+  ? []
+  : T extends Array<any> | string
+  ? string[]
+  : never
+
+interface ObjectConstructor {
+  keys<T>(o: T): ObjectKeys<T>
+}
```

Comment:
Where does we need this type?

Reply:
This type is used wherever we use `Object.keys`. By default, the type of `keys` is `string[]`, but it should more literally be `keyof typeof myObject`. Doing `Object.keys(myObject).map(key => myObject[key])` may throw a TS warning since you can't index `myObject` with `string`, and this interface resolves that.

Reply:
Makes a sense!